src/renderer/entities/operation/ui/SignButton.tsx

```diff
@@ -37,14 +31,18 @@ const WalletText: Record<WalletType, string> = {
 export const SignButton = ({ disabled, type, onClick, className }: Props) => {
   const { t } = useI18n();
 
+  const isSignableType = type in WalletIcon;
+
+  if (!isSignableType) return <></>;
```

Comment:
I guess this is enough, `isSignableType` is not used anywhere else
```suggestion
  if (!(type in WalletIcon)) return null;
```